backend-rust/CHANGELOG.md

```diff
@@ -43,6 +43,45 @@ Modifying search key on the following tables to all be using `text_pattern_ops`
 - Fix baker transactions pagination
 - Fix baker delegation pagination to be sorted by stake as primary instead of using account index.
 
+### Changed
```

Comment:
please add to the changelog what you have added to the  `backend-rust` project instead. E.g. that there exists a dummy server for plt support and which new endpoints/queries are now exposed at the backend.

Reply:
Added a new change log for plt dummy queries. please review.